sql/core/src/test/resources/sql-tests/inputs/operators.sql

```diff
@@ -92,3 +91,7 @@ select abs(-3.13), abs('-2.19');
 
 -- positive/negative
 select positive('-1.11'), positive(-1.11), negative('-1.11'), negative(-1.11);
+
+-- trunc number
+select trunc(1234567891.1234567891, 4), trunc(1234567891.1234567891, -4), trunc(1234567891.1234567891, 0), trunc(1234567891.1234567891);
+select trunc(1234567891.1234567891, null), trunc(null, 4), trunc(null, null);
```

Comment:
Can you add test cases like `trunc(1234567891.1234567891, "yyyy")` and `trunc('2015-07-22', 4)`?